src/modules/inspector/sagas.ts

```diff
@@ -291,8 +298,6 @@ export function* inspectorSaga(store: RootStore) {
 
     const scene: SceneSDK7 = yield getScene()
 
-    FILES.delete(path)
```

Comment:
should we remove it from memory at least? this delete is expected to delete it from the scene only or its supposed to delete the uploaded file too?